src/containers/BooksList.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/jsx-key */
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+
```

Comment:
this component should connect to the redux store [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/ConnetToRedux.png)]
Something like this:
```
import { connect } from "react-redux";
```

Reply:
code above is to connect, then now
use `mapStateToProps` to fetch books from the `store`